examples/five_pl_logistic/five_pl_fit.ipynb

```diff
@@ -0,0 +1,934 @@
+       "               C=0.6903113152785875, D=3.68945557111049, S=0.20269883209762796)</pre><b>In a Jupyter environment, please rerun this cell to show the HTML representation or trust the notebook. <br />On GitHub, the HTML representation is unable to render, please try loading this page with nbviewer.org.</b></div><div class=\"sk-container\" hidden><div class=\"sk-item\"><div class=\"sk-estimator sk-toggleable\"><input class=\"sk-toggleable__control sk-hidden--visually\" id=\"sk-estimator-id-12\" type=\"checkbox\" checked><label for=\"sk-estimator-id-12\" class=\"sk-toggleable__label sk-toggleable__label-arrow\">FivePLLogistic</label><div class=\"sk-toggleable__content\"><pre>FivePLLogistic(A=0.002006901257671911, B=3.087036820144977,\n",
+       "               C=0.6903113152785875, D=3.68945557111049, S=0.20269883209762796)</pre></div></div></div></div></div>"
+      ]
+     },
+     "execution_count": 38,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 38
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "Note that when we performed the fit, we provided a weight function based on the magnitude of observed values. This weight scheme ensures our model is not biased towards high signal values--this is especially important because our data spans several orders of magnitude. You can provide your own preferred weight function during fitting.\n"
```

Comment:
Should use the `inverse_variance_weight_function` method defined in bio-curve-fit instead of this `lambda x:x`. It's unclear that `lambda x:x` is the correct function without looking under the hood of scipy.optimize or reading the docstring for the fit function...all details that distract a bit from the basic explanation going on here